st2common/setup.py

```diff
@@ -62,5 +62,10 @@
         'bin/st2-track-result',
         'bin/st2-validate-pack-config',
         'bin/st2-check-license'
-    ]
+    ],
+    entry_points={
+        'st2common.metrics.driver': [
+            'statsd = st2common.metrics.drivers.statsd_driver:StatsdDriver',
```

Comment:
Entry point for prometheus?  I assume you will add this once we finalize on the plugin implementation.

Reply:
Yes, I need to do some more digging into the prometheus integration before I'm ready to register it my default.

Reply:
Can we add a noop metrics driver here?